### ISSUE TITLE
Can't install PSA CONTROLLER

### ISSUE THREAD

**Issue by solproget** [flobz/psa_car_controller#600]
I can't install the addon.
Here's the error it gives me

Can't install flobz/psa_car_controller:v3.0.12: 404 Client Error for http+docker://localhost/v1.42/images/flobz/psa_car_controller:v3.0.12/json: Not Found ("No such image: flobz/psa_car_controller:v3.0.12")

Can anyone help me solve?
Thanks for the help
![PSA](https://github.com/flobz/psa_car_controller/assets/77103958/7f8c3427-2b8e-4405-b341-ba76a64cd899)

**Comment by JS-Lorthe** [comment]
Hello,

Same issue. I have restart my Raspberry and HA but same result. Is it compatible with HA OS (not docker or installation on Linux distribution)?

Thank you

**Comment by flobz** [comment]
What is the result of this commands: 
```
docker pull flobz/psa_car_controller:v3.0.12
```

**Comment by solproget** [comment]
> What is the result of this commands:
> 
> ```
> docker pull flobz/psa_car_controller:v3.0.12
> ```

I used the command and found that I needed to disable Protect Mode. Now I can install.
In the "Advanced SSH & Web Terminal" add-on I disabled it but now it tells me:
"Security mode is disabled!
Security mode on this add-on is disabled! This gives the add-on full access to the entire system, which adds security risks and could harm your system if used incorrectly. Disable Safe Mode only if you know, need, and trust the source of this add-on."
Can I rest assured?
Thanks for the help
![SSH](https://github.com/flobz/psa_car_controller/assets/77103958/6473fe77-a8e7-4445-8c50-174474b56d4c)
![CAR](https://github.com/flobz/psa_car_controller/assets/77103958/682ac0e4-6a00-4f1f-b28c-8a84adff8128)
